content/ru/docs/concepts/workloads/controllers/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Ресурсы рабочей нагрузки"
+weight: 20
+simple_list: true
+---
+
+Kubernetes предоставляет несколько встроенных API для декларативного управления
+{{< glossary_tooltip text="рабочими нагрузками" term_id="workload" >}}
+и их компонентами.
+
+В конечном счете, приложения работают в качестве контейнеров внутри
+{{< glossary_tooltip term_id="Pod" text="подов" >}}; однако, управление отдельными подами требует больших усилий.
+Например, если под выходит из строя, вам, вероятно, надо будет
+запустить новый под, чтобы заменить его. Kubernetes может сделать это за вас.
+
+Вы можете использовать Kubernetes API для создания 
+{{< glossary_tooltip text="объекта" term_id="object" >}} рабочей нагрузки, который представляет собой более высокий уровень абстракции, чем под, и затем {{< glossary_tooltip text="слой управления" term_id="control-plane" >}} Kubernetes
+автоматически управляет подами за вас, на основе спецификации объекта рабочей нагрузки.
+
+Встроенные API для управления рабочими нагрузками: 
+
+Деплоймент ([Deployment](/docs/concepts/workloads/controllers/deployment/)) (и, косвенно, [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/)), - это наиболее распространенный способ запуска приложения в кластере.
+Deployment хорошо подходит для управления неизменной (stateless) рабочей нагрузкой в кластере, где
+любой под в деплойменте не содержит изменяемых данных и может быть заменен при необходимости
+(Deployments — замена устаревшего ресурса
+{{< glossary_tooltip text="ReplicationController" term_id="replication-controller" >}} API).
+
+[StatefulSet](/docs/concepts/workloads/controllers/statefulset/) позволяет управлять одним или несколькими подами,
+в которых работает одно и то же приложение, и где поды имеют четкую идентичность (являются stateful).
```

Comment:
That's a super tricky phrase to translate both elegantly and correctly. My suggestion here is just a suggestion:
```suggestion
в которых работает одно и то же приложение, для случаев, когда все эти поды требуют четкой идентичности (т.е. являются stateful).
```